### PR TITLE
Use k8s podname as worker id.

### PR DESCRIPTION
**Pull request by tomprince**

### CODE REVIEW

**Comment by tomprince** [comment]
This depend on mozilla-services/cloudops-infra#2386

File: configloader/src/configloader/script.py [hunk 1]
```diff
@@ -9,11 +9,26 @@
 import yaml
 
 
+def generate_worker_id(pod_name):
+    """
+    Generate a worker id based on the pod name it is running in.
+
+    Since the worke id is limited to 38 characters, we slice off `-` delimited
```
Comment (rail) [inline configloader/src/configloader/script.py]: A typo: s/worke/worker/

**Review by rail — APPROVED** [review]

**Comment by rail** [comment]
The CI failure is an easy style fix.

**Comment by bhearsum** [comment]
The cloudops change is done: https://github.com/mozilla-services/cloudops-infra/pull/3251

I'll push this to a dev worker at some point.

**Comment by bhearsum** [comment]
I tried this out on dev today and discovered that it requires some ci-config changes (for scopes) to work properly. It's not worth spending any more time on IMO, so I'm closing it out.

**Pull request closed without merging**